### ISSUE TITLE
webpdf's use of /tmp breaks markdown images

### ISSUE THREAD

**Issue by ryanlovett** [jupyter/nbconvert#1938]
If a user specificies a relative path to an image, e.g. `![myimage](images/myimage.png)`, the image display correctly in the jupyter app, but does not get embedded in the PDF after conversion from webpdf. If the user specifies an absolute path to the image, it does not display correctly in the jupyterap, but does get embedded in the PDF.

I suspect this is because the webpdf conversion writes out a temporary html file to /tmp/. The HTML files looks for relative paths there rather than the user's currect working directly. When I manually images to /tmp (e.g. `/tmp/images/myimage.png`), with my notebook in `/home/jovyan` and using a relative markdown path of `images/myimage.png`, the image is embedded in the PDF properly.

**Comment by tuncbkose** [comment]
> If the user specifies an absolute path to the image, it does not display correctly in the jupyterap

This seems to be intended behavior, https://github.com/jupyterlab/jupyterlab/issues/6835.

I agree that the rest should be fixed.

**Comment by normanius** [comment]
This is still an issue, unfortunately. 

For context, this is output generated with.

`jupyter nbconverter --to pdfviahtml path/to/notebook.ipynb`

I generally like the output that is achieved this way (math typesetting look great, svg graphics are displayed nicely). Also, I like the custom-format, single-page PDF since too many page-breaks break readability IMHO.

However, the fact that images are excluded make it for me not usable.

**Comment by JPFrancoia** [comment]
Yes, same problem here. The solution give here kinda works: https://github.com/jupyter/nbconvert/issues/1079#issuecomment-1012601618 and I'm able to get images in the pdf generated with pdfviahtml. But it's a bit hackish and the code used to display the image is visible in the output

**Comment by soilstack** [comment]
This kind of kills using nbonvert to make pdfs ....  Suprised it's not been fixed somehow.

**Comment by Pear-sudo** [comment]
Same issue here. It’s quite annoying to have multiple images missing in the PDF.

**Comment by Albresky** [comment]
I encounted the same issue that embeded images in Markdown cells with relative path could not exported in webPDF. 

**Solution:**
I succeeded in exporting PDF using webPDF with embeded images in Markdown cells displaying fine through **changing the path of images in Markdown cells into absolute path**, e.g. do not use `![figure](./foo.png)`, use `![figure](/home/foo_user/project_dir/foo.png)` instead.

It's not elegant, but it works!
